test/cpp/codegen/compiler_test_mock_golden

```diff
@@ -2,6 +2,9 @@
 // If you make any local change, they will be lost.
 // source: src/proto/grpc/testing/compiler_test.proto
 
+#ifndef GRPC_MOCK_src_2fproto_2fgrpc_2ftesting_2fcompiler_5ftest_2eproto__INCLUDED
```

Comment:
maybe remove the 2f, 5f and 2e characters

Reply:
This is how grpc codegen doe; `_2f` = `/`, `_5f` = `_`, and so on.